README.md

```diff
@@ -67,3 +67,34 @@ to handle output from:
 * Quantum Mechanics applications, such as Jaguar
 * Protein-Protein Docking applications
 * Many other backends used in both Life and Material Sciences
+
+Installation
+============
+
+Command line installation on a
+[Unix-like](https://en.wikipedia.org/wiki/Unix-like)
+operating system follows a typical configure, build, and test procedure.
+Configuration is via [CMake](https://en.wikipedia.org/wiki/CMake);
+building is via [Make](https://en.wikipedia.org/wiki/Make_(software));
+and testing can be performed by running a custom built program.  Here is an
+example command sequence:
+
+```bash
+git clone git@github.com:schrodinger/maeparser.git maeparser
+cd maeparser
+mkdir build
+cd build
+export CC=gcc
+export VERBOSE=yes
+cmake `pwd/..`
+make
+test/unittest 
+```
```

Comment:
```suggestion
```bash
# "clone" a local copy of the code from github
$ git clone git@github.com:schrodinger/maeparser.git maeparser

# set up the build configuration
$ cd maeparser
$ mkdir build
$ cd build
$ export CC=gcc
$ export VERBOSE=yes
$ cmake `pwd/..`

# Run the build
$ make

# Run the tests
$ test/unittest 
```
```